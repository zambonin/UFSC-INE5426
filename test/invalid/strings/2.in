# more than one character inside simple quotes
char a = 'as'
