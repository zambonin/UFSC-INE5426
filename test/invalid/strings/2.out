[Line 2] warning: value truncated to 'a'
